saleor/product/models.py

```diff
@@ -185,6 +185,11 @@ class ProductVariant(models.Model):
         Product, related_name='variants', on_delete=models.CASCADE)
     attributes = HStoreField(default={}, blank=True)
     images = models.ManyToManyField('ProductImage', through='VariantImage')
+    handle_stock = models.BooleanField(
```

Comment:
Although I understood the purpose of this field from reading the code, this name sounds a bit too general to me. What about `track_inventory`? I think it would be more understandable.